pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -604,6 +759,26 @@ func (lbaas *LbaasV2) UpdateLoadBalancer(clusterName string, service *api.Servic
 			if err != nil && !isNotFound(err) {
 				return err
 			}
+			// Delete the rules in the Node Security Group
```

Comment:
Why is this here?
We're removing a backend host from the LB pool, but I don't see anything node-specific about the secgroup rule you're deleting...